<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mxr_renamer_window</class>
 <widget class="QMainWindow" name="mxr_renamer_window">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>405</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>291</width>
    <height>405</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>291</width>
    <height>405</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>3DS Renamer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>281</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="title">
     <string>RENAME</string>
    </property>
    <widget class="QCheckBox" name="chk_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>119</width>
       <height>17</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>New Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_name">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>174</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>80</number>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>itemname</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_prefix">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>Prefix</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_prefix">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="maxLength">
      <number>80</number>
     </property>
     <property name="placeholderText">
      <string>prefix</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_suffix">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>Suffix</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_suffix">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="placeholderText">
      <string>suffix</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_removefirst">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>Remove First X Digit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_removefirst">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_removelast">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>Remove Last X Digit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_removelast">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="maxLength">
      <number>80</number>
     </property>
     <property name="cursorPosition">
      <number>1</number>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_rename">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>RENAME</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_use_padding">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>Use Padding</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_basenumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>Base Number</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_basenumber">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_step">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>Increment</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_padding_step">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="maxLength">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_padding_zeros">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>270</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="title">
     <string>FIND / REPLACE</string>
    </property>
    <widget class="QLineEdit" name="line_find">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>80</number>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>findText</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_replace">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>REPLACE</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>Replace</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_replace">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>80</number>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>replaceText</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rad_selection">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>Selection</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rad_scene">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>Scene</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="mnu_underscore">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic underscores</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btn_rename</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
